docker/kubernetes/kustomize/README.md

```diff
@@ -0,0 +1,34 @@
+
+# Kubernetes Deployment Files
+
```

Comment:
I would add a prerequisite section here that link to how to install kustomize and set up kustomize.
This is not something everyone knows how to do.

Reply:
I added a note for kustomize and argo-cd